.php_cs.dist

```diff
@@ -12,6 +12,7 @@ return PhpCsFixer\Config::create()
         'php_unit_no_expectation_annotation' => false, // part of `PHPUnitXYMigration:risky` ruleset, to be enabled when PHPUnit 4.x support will be dropped, as we don't want to rewrite exceptions handling twice
         'array_syntax' => ['syntax' => 'short'],
         'fopen_flags' => false,
+        'orderer_class_elements' => true,
```

Comment:
you mean `ordered_class_elements`?

Reply:
Yes exactly sorry about that typo error I've commited the correct rule's name